src/components/CookiesBanner/index.tsx

```diff
@@ -163,9 +163,9 @@ const CookiesBanner = (): ReactElement => {
 
     // we remove GA cookies manually as react-ga does not provides a utility for it.
     if (!localAnalytics) {
-      removeCookie('_ga')
-      removeCookie('_gat')
-      removeCookie('_gid')
+      removeCookie(COOKIE_GA.name, COOKIE_GA.path)
+      removeCookie(COOKIE_GAT.name, COOKIE_GAT.path)
+      removeCookie(COOKIE_GIT.name, COOKIE_GIT.path)
```

Comment:
Same question, can we use an array map here?

Reply:
you are seeing an old code.